config.md

```diff
@@ -118,6 +118,10 @@ For more information about SELinux, see  [Selinux documentation](http://selinuxp
 * **`noNewPrivileges`** (bool, optional) setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
 [The kernel doc](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt) has more information on how this is achieved using a prctl system call.
 
+For Windows-based systems, the process structure supports the following process specific fields:
+
+* **`initialconsolesize`** (array of ints, optional) specifies the initial console size (height, width) of the terminal if attached.
```

Comment:
Yes, I deliberately did this as the capitalisation is consistent with other items such as rlimits (vs `rLimits`).  Are those incorrect as well? If so, I'll open another PR to address that, but fix the capitalisation to `initialConsoleSize` here.